content/swift/concepts/dictionaries/terms/keys/keys.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.keys()'
+Description: 'Returns a collection containing the keys of a dictionary.'
+Subjects:
+  - 'Computer Science'
+  - 'iOS'
+  - 'Code Foundations'
+Tags:
+  - 'Swift'
+  - 'Dictionary'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+In Swift, the `keys` property of a dictionary returns an unordered collection containing all the keys within the dictionary. Since this collection is unordered, the order of the keys may vary each time it is accessed. This property is part of the `Dictionary` structure and is useful when iterating through or manipulating just the keys of a dictionary.
+
+## Syntax
+
+```swift
+dictionaryName.keys
+```
+
+- `dictionaryName`: The instance of a `Dictionary` whose keys you want to access.
+- The `keys` property returns a `Dictionary.Keys` collection, which contains all the keys in the dictionary.
+
+> **Note:** The `keys` property returns a collection, not an array, but it can be converted to an array if needed using [`Array`](https://www.codecademy.com/resources/docs/swift/arrays)(dictionary.keys).
```

Comment:
```suggestion
> **Note:** The `keys` property returns a collection, not an array, but it can be converted to an array if needed using [Array](https://www.codecademy.com/resources/docs/swift/arrays).
```